epochStart/bootstrap/process.go

```diff
@@ -836,12 +841,36 @@ func (e *epochStartBootstrap) createTriesComponentsForShardId(shardId uint32) er
 		return err
 	}
 
+	err = e.tryCloseExisting(factory.PeerAccountTrie)
+	if err != nil {
+		return err
+	}
+
 	e.trieContainer.Put([]byte(factory.PeerAccountTrie), peerAccountsTrie)
 	e.trieStorageManagers[factory.PeerAccountTrie] = peerStorageManager
 
 	return nil
 }
 
+func (e *epochStartBootstrap) tryCloseExisting(trieType string) error {
+	var err error
+	existingTrie := e.trieContainer.Get([]byte(trieType))
+	if existingTrie != nil {
+		err = existingTrie.ClosePersister()
+		if err != nil {
```

Comment:
I do not think this is too critical. Maybe just log with error? valid also for L868

Reply:
added log.warn